css/debug-bar.dev.css

```diff
@@ -221,6 +221,30 @@ body.debug-bar-maximized.debug-bar-visible {
 	background: rgba( 0, 0, 0, 0.2 );
 }
 
+#debug-menu-links li span.debug-bar-issue-count {
```

Comment:
Is there any way all of these CSS selectors could be made less specific? These are really hard to override